### ISSUE TITLE
Feature Request - License check not be possible to succeed for TEMP licences except for categories that allow TEMP

### ISSUE THREAD

**Issue by stuartlynne** [esitarski/RaceDB#67]
Here (BC, Canada) we require a valid UCI or equivalent license for all categories EXCEPT novice which can be entered with a single-day event license.

Currently, RaceDB appears to allow the License Check to pass when the rider has a TEMP license. It probably should not. Or at the very least should pop up a dialogue saying don't do that. Then another dialogue saying are you sure you want to do that.

In other words make it more painful to skip the warnings than simply correct it. 

Possibly having a spot in the license check screen to update the license number would help.
